.github/workflows/tests-e2e-npm.yml

```diff
@@ -75,8 +79,9 @@ jobs:
         if: always()
         with:
           name: playwright-report
-          path: ${{ runner.temp }}/playwright-report/
+          path: ./playwright-report/
           retention-days: 90
+          if-no-files-found: error
       - name: Publish CTRF Test Summary Results
-        run: npx github-actions-ctrf ${{ runner.temp }}/playwright-report/ctrf-report.json
+        run: npx github-actions-ctrf ./playwright-report/ctrf-report.json || exit 1
```

Comment:
I don't see quite why this helps. Going to do some digging. I still think the output will be getting written to `runner.temp`

Reply:
comment above addresses the same